Project.toml

```
@@ -12,12 +12,12 @@ Requires = "ae029012-a4dd-5104-9daa-d747884805df"
 TimeZones = "f269a46b-ccf7-5d73-abea-4c690281aa53"
 
 [compat]
-CategoricalArrays = "≥ 0.3.0"
-CodecZlib = "≥ 0.4.0"
-DataFrames = "≥ 0.19.0"
-FileIO = "≥ 1.0.5"
-TimeZones = "≥ 0.7.3"
-julia = "1.0"
+CategoricalArrays = "0.5, 0.6, 0.7"
+CodecZlib = "0.4, 0.5, 0.6"
+DataFrames = "0.19, 0.20"
+FileIO = "1.0.5, 1.1, 1.2"
```

This is enough to allow versions up to 2.0:
```suggestion
FileIO = "1.0.5"
```

---

oof.  right.  but my reading of the [docs](https://julialang.github.io/Pkg.jl/v1/compatibility/#Caret-specifiers-1) makes me think that "1.0.5", which implicitly has a "^" prefixed, would also yield [1.0.5,2.0.0).  is what we really want here "~1.0.5, ~1.1, ~1.2"?

---

That's what we want. No version should be breaking until 2.0.

---

i've now added tilde's to FileIO.  anything else?

---

Why use tildes? The most simple syntax is generally the appropriate one (that's why it's the default). So my suggestion still holds:
```suggestion
FileIO = "1.0.5"
```

---

sorry, i misunderstood "that's what we want" to mean the tildes.  i've made it just 1.0.5.